src/main/java/io/lettuce/core/cluster/pubsub/RedisClusterPubSubListener.java

```diff
@@ -67,4 +67,16 @@ public interface RedisClusterPubSubListener<K, V> {
      */
     void punsubscribed(RedisClusterNode node, K pattern, long count);
 
+    /**
+     * Subscribed to a shard channel.
+     *
+     * @param node the {@link RedisClusterNode} from which the {@code message} originates.
+     * @param shardChannel Shard channel
+     * @param count Subscription count.
+     */
+    default void ssubscribed(RedisClusterNode node, K shardChannel, long count){
```

Comment:
Please add `@since 7.0` to indicate since which version this method was introduced.

Reply:
done